benchmark/test_biocypher_query_generation.py

```diff
@@ -140,15 +150,300 @@ def test_end_to_end_query_generation(prompt_engine):
     score.append("Gene" in query)
     score.append("Disease" in query)
     score.append("mucoviscidosis" in query)
+    cypher_regex = r"MATCH \([a-zA-Z]*:Gene\)<-\[[a-zA-Z]*:PERTURBED\]-\([a-zA-Z]*:Disease.*\)|MATCH \([a-zA-Z]*:Disease.*\)-\[[a-zA-Z]*:PERTURBED\]->\([a-zA-Z]*:Gene\)"
+    score.append((re.search(cypher_regex, query) is not None))
+    score.append("WHERE" in query or "{name:" in query)
+
+    with open(FILE_PATH, "a") as f:
+        f.write(
+            f"{prompt_engine.model_name},end-to-end_single,{calculate_test_score(score)}\n"
+        )
+
+
+######
+# test multiple word entities
+######
+
+
+# entity selection doesn't have any issue
```

Comment:
Fine for now, but I would not expect this to hold for all models. In the context of the benchmark across models, prompts, etc, the scores will be the measure of whether it works or not.

Reply:
Removed this comment after clean up